devtools/cli/src/main/java/io/quarkus/cli/common/BuildOptions.java

```diff
@@ -13,7 +13,7 @@
     @CommandLine.Option(order = 5, names = { "--offline" }, description = "Work offline.", defaultValue = "false")
     public boolean offline = false;
 
-    @CommandLine.Option(order = 6, names = { "--tests" }, description = "Run tests.", negatable = true)
+    @CommandLine.Option(order = 6, names = { "--skip-tests" }, description = "Skip tests.")
     public boolean runTests = true;
```

Comment:
Shouldn't the meaning of this property be changed too? I suspect it won't work as is (or I might miss how Picocli is working as I have no experience with it).

Reply:
I don't agree with this change. I am on PTO, but I would prefer to make the options as they exist work (which I thought I had tests for, but apparently not...).